packages/cloud-cognitive/src/components/ComboButton/ComboButton.stories.js

```diff
@@ -17,19 +17,60 @@ import { ComboButton, ComboButtonItem } from '..';
 
 import styles from './_combo-button.scss';
 
+const defaultArgs = {
+  size: 'md',
+};
+
 export default {
   title: getStoryTitle(ComboButton.displayName),
   component: ComboButton,
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+      },
+      options: ['sm', 'md', 'lg'],
+    },
+  },
   subcomponents: {
     ComboButtonItem,
   },
   parameters: { styles },
 };
 
-export const Default = prepareStory(() => (
-  <ComboButton>
-    <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
-    <ComboButtonItem renderIcon={CloudApp16}>ComboButtonItem 2</ComboButtonItem>
-    <ComboButtonItem>ComboButtonItem 3</ComboButtonItem>
-  </ComboButton>
-));
+const Template = (args) => {
+  return (
+    <ComboButton {...args}>
+      <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
+      <ComboButtonItem renderIcon={args.icon && args.icon}>
+        ComboButtonItem 2
+      </ComboButtonItem>
+      <ComboButtonItem>ComboButtonItem 3</ComboButtonItem>
+      <ComboButtonItem hasDivider={args.hasDivider}>
```

Comment:
It might make more sense to use something like `<ComboButtonDivider/>`, similar to how Carbon does in its SideNav.

Example from Carbon:
![image](https://user-images.githubusercontent.com/1717398/184219382-9525dcc1-706a-43f0-a219-0950ebd5d9b8.png)

While I have seen `hasDivider`, I think usually, that’s for dividers in a fixed position. In the case of the combo button, the consumer/should have more control over where example the divider is placed, in which case the divider subcomponent might be the better option.